runtime/tcore.c

```diff
@@ -133,15 +134,24 @@ TString *titan_string_concatN(lua_State *L, size_t n, TString **ss)
     }
 }
 
+// These definitions are from ltable.h:
```

Comment:
1. Could we keep the C style comments like the generated code?
2. s/ltable.h/ltable.c/
3. I would remove the `:` at EOL.